docs/downstream/odh-install-crc.md

```diff
@@ -10,12 +10,11 @@ oc get packagemanifests -n openshift-marketplace | grep opendatahub-operator
 
 If it's not present, then you need to troubleshoot the marketplace: https://github.com/operator-framework/operator-marketplace/issues/344
 
-
-Then we need to create a projects named `odh-operator` and `odh-deployment`. (The name of the project is hard-coded in the `kustomize` files in https://github.com/anishasthana/my-odh-install and you can change is there using `kustomize`.)
+Then we need to create a projects named `odh-operator` and `opf-*`. (The name of the project is hard-coded in the `kustomize` files in https://github.com/anishasthana/my-odh-install and you can change is there using `kustomize`.)
```

Comment:
Can we update this to point at https://github.com/operate-first/odh instead?

Reply:
sure thing! :slightly_smiling_face: